developer-handbook/github-process/post-code-review.md

```diff
@@ -0,0 +1,19 @@
+# Post Code Review
+
+Your Pull Request was reviewed by a colleague (or a few colleagues, which is great). What's left to do now is:
+
+## Address it
+- Just like in the code review step, we opt for friendly tones in their replies as well! We all have the best intentions for our codebases and every input is valuable.
+
+- React or reply to every comment that was left, even if it's just "acknowledged" or "done", or an emoji reaction. The person that left it thought it was worth saying something, it's nice to acknowledge it.
+
+- The owner of the PR can "resolve" comments after commenting "done" if they are small things like typos. For bigger changes than that, the responsibility of resolving should be passed on the the reviewer as they can use it to confirm the changes are inline with what they'd suggested.
```

Comment:
'passed on the the reviewer' should be 'passed on to the reviewer' I think

Reply:
Addressed, resolving.